Gemfile

```diff
@@ -43,7 +43,7 @@ group :test do
   gem 'capybara'
   gem 'ci_reporter'
   gem 'govuk-content-schema-test-helpers'
-  gem 'govuk_schemas'
+  gem 'govuk_schemas', '2.3.0'
```

Comment:
What's the reasoning behind adding this restriction?

Reply:
There is a breaking change in version 3.0 - to avoid doing too much inside this PR I've locked it to the last version before 3.0